src/DataViews/FunctionsDataView.cpp

```diff
@@ -36,8 +36,11 @@ FunctionsDataView::FunctionsDataView(AppInterface* app,
     : DataView(DataViewType::kFunctions, app, metrics_uploader) {}
 
 const std::string FunctionsDataView::kUnselectedFunctionString = "";
-const std::string FunctionsDataView::kSelectedFunctionString = "✓";
+const std::string FunctionsDataView::kSelectedFunctionString = "H";
```

Comment:
I am wondering if we should use the opportunity to introduce global constants for those, so that we can use them in all data views?

Reply:
In this PR?

Reply:
Yeah, let's make it in this PR.